src/lib/updating-element.ts

```diff
@@ -191,6 +191,13 @@ type UpdateState = typeof STATE_HAS_UPDATED|typeof STATE_UPDATE_REQUESTED|
     typeof STATE_IS_REFLECTING_TO_ATTRIBUTE|
     typeof STATE_IS_REFLECTING_TO_PROPERTY|typeof STATE_HAS_CONNECTED;
 
+/**
+ * The Closure JS Compiler doesn't currently have good support for static
+ * propertiy semantics where "this" is dynamic, so we use this hack to bypass
+ * any rewriting by the compiler.
+ */
+const finalized = '__finalized__';
```

Comment:
Hm, if we're going this approach, I think we could do something like the following:

```typescript
const finalized = 'finalized';

class UpdateElement {
  protected static [finalized] = false;
  protected static finalize() {
    this[finalized] = true;
    // ...
  }
}
```

Reply:
Done.